File: 1791-richest-customer-wealth/1791-richest-customer-wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        a=[]
        n = len(accounts)
        
        for i in range(n):
            sum = 0
            for j in range(len(accounts[i])):
                sum += accounts[i][j]
            a.append(sum)
        return max(a)
            
        
      
        

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& a) {
        vector<int> v;
        int n = a.size();
        for(int i=0;i<n;i++){
            int sum = 0;
            for (int j=0;j<a[i].size();j++){
                sum = sum+a[i][j];
            }
             v.push_back(sum);
        }
        return *max_element(v.begin(),v.end());        
    }
};